### PR TITLE
Add a doc with JetStream troubleshooting hints

### PR DESCRIPTION
**Pull request by rodrigc**
I put this doc together based on feedback in these Slack threads:
https://natsio.slack.com/archives/C069GSYFP/p1665766924057909?thread_ts=1665764019.736979&cid=C069GSYFP
https://natsio.slack.com/archives/CM3T6T7JQ/p1665656371076639?thread_ts=1665655333.400419&cid=CM3T6T7JQ

The doc fully rendered is here:
https://github.com/nats-io/nats.docs/blob/b6199a6f92ac8826afb08c3d0bd069f4d0d18007/running-a-nats-service/configuration/clustering/jetstream_clustering/troubleshooting.md

### CODE REVIEW

**Comment by gcolliso** [comment]
Thanks @rodrigc! 
@jnmoyne or @codegangsta could you review and provide comments and/or approval?

File: running-a-nats-service/configuration/clustering/jetstream_clustering/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Troubleshooting NATS JetStream clusters
+
+Diagnosing problems in NATS JetStream clusters requires:
+* knowledge of [JetStream concepts](/nats-concepts/jetstream/#jetstream-1)
```
Comment (bruth) [inline running-a-nats-service/configuration/clustering/jetstream_clustering/troubleshooting.md:4]:
```suggestion
* knowledge of [JetStream concepts](../../../../nats-concepts/jetstream/readme.md)
```

File: running-a-nats-service/configuration/clustering/jetstream_clustering/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+## Troubleshooting tips
+
+1. Look at [nats-server](https://github.com/nats-io/nats-server#readme) logs
+2. Make sure that in the [NATS JetStream configuration](./README.md#configuration), at least one system user is configured in this section: `{ $SYS { users } }`.
+
+### `nats account` commands
+
+| Command | Description |
+| :--- | :--- |
+| [nats account info](../../../nats_admin/jetstream_admin/account.md) | Verify that JetStream is enabled on account |
```
Comment (bruth) [inline running-a-nats-service/configuration/clustering/jetstream_clustering/troubleshooting.md:18]:
```suggestion
| [`nats account info`](../../../nats_admin/jetstream_admin/account.md) | Verify that JetStream is enabled on account |
```

File: running-a-nats-service/configuration/clustering/jetstream_clustering/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+###  Basic `nats server` commands
+
+| Command | Description |
+| :--- | :--- |
+| `nats server ls` | List known servers |
+|  `nats server ping`  |    Ping all servers |
+|  `nats server info`  |    Show information about a single server |
+|  [nats server check](../../../clients.md#testing-your-setup) | Health check for NATS servers |
```
Comment (bruth) [inline running-a-nats-service/configuration/clustering/jetstream_clustering/troubleshooting.md:27]:
```suggestion
|  [`nats server check`](../../../clients.md#testing-your-setup) | Health check for NATS servers |
```

File: running-a-nats-service/configuration/clustering/jetstream_clustering/troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+### `nats server report` commands
+
+| Command | Description |
+| :--- | :--- |
+| `nats server report connections` |  Report on connections |
+| `nats server report accounts` | Report on account activity |
+| [nats server report jetstream](./administration.md#viewing-the-cluster-state) | Report on JetStream activity |
```
Comment (bruth) [inline running-a-nats-service/configuration/clustering/jetstream_clustering/troubleshooting.md:35]:
```suggestion
| [`nats server report jetstream`](./administration.md#viewing-the-cluster-state) | Report on JetStream activity |
```

File: running-a-nats-service/configuration/clustering/jetstream_clustering/troubleshooting.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+### `nats server request` commands
+
+| Command | Description |
+| :--- | :--- |
+|  [nats server request jetstream](./administration.md#viewing-the-cluster-state) | Show JetStream details |
```
Comment (bruth) [inline running-a-nats-service/configuration/clustering/jetstream_clustering/troubleshooting.md:41]:
```suggestion
|  [`nats server request jetstream`](./administration.md#viewing-the-cluster-state) | Show JetStream details |
```

**Review by bruth — CHANGES_REQUESTED** [review]
Thanks for putting this together! A few minor adjustments, but otherwise looks good.

**Comment by rodrigc** [comment]
@bruth I incorporated all your suggestions

**Comment by jnmoyne** [comment]
LGTM

**Review by jnmoyne — APPROVED** [review]

**Review by bruth — APPROVED** [review]

**Pull request merged**